install/linux/usr/share/odemis/hwtest/pmd401.piezomotor.tsv

```diff
@@ -0,0 +1,19 @@
+# Parameters for PMD301/PMD401
+# Bank/Axis	Address	Value	Description
```

Comment:
I don't think there is such "bank" notion in the controller, so let's remove any reference to banks from this file, and the parser function.